go.mod

```diff
@@ -393,7 +393,8 @@ replace github.com/eclipse/paho.mqtt.golang => github.com/shivamkm07/paho.mqtt.g
 // Uncomment for local development for testing with changes in the components-contrib repository.
 // Don't commit with this uncommented!
 //
-// replace github.com/dapr/components-contrib => ../components-contrib
+replace github.com/dapr/components-contrib => github.com/DeepanshuA/components-contrib v1.6.0-rc.2.0.20220912174448-d1d3384d6575
```

Comment:
Need to revert this change after PR introducing bulk pubsub interfaces is merged

Reply:
Ok for using a temp replace, but please keep it at the bottom and don't replace this line, which is a hint to contributors for using a local clone.

Reply:
Plan to remove this reference - as first https://github.com/dapr/components-contrib/pull/2024 would be merged.